app/assets/stylesheets/_screen.scss

```diff
@@ -653,9 +653,9 @@ div#keycut-help {
   }
 }
 
-.projects {
-  display: flex;
-  flex-wrap: wrap;
+.project-title{
+  display: block;
+  line-height: 12px;
```

Comment:
It's a good practice not to use units when defining a value for the `line-height` property, it can lead to some weird behavior with children elements. https://css-tricks.com/almanac/properties/l/line-height/